workflows/genome-assembly/polish-with-long-reads/.dockstore.yml

```diff
@@ -0,0 +1,11 @@
+version: 1.2
+workflows:
+- name: main
+  subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /Assembly-polishing-with-long-reads.ga
```

Comment:
You'd have to update this as well

Reply:
Should I update the version?

Reply:
No, it seems your workflow is called `Assembly_polishing_with_long_reads.ga` now, you either need to rename the workflow to `Assembly-polishing-with-long-reads.ga` or change this to `Assembly_polishing_with_long_reads.ga`. I think my preference would be to rename the workflow back to `Assembly-polishing-with-long-reads.ga`